### PR TITLE
[Windows] Add manual package mgt cases

### PR DESCRIPTION
**Pull request by zhuyongyong**
Impacted tests(approved): new 6, update 0, delete 0
Unit test platform: Crosswalk Project for Windows 18.46.457.0
Unit test result summary: pass 4, fail 2, block 0

https://crosswalk-project.org/jira/browse/XWALK-5983

### CODE REVIEW

**Comment by jennycao** [comment]
@Honry Pls also help to review

File: wrt/wrt-packagemgtmanu-android-tests/packagemgtmanu/Crosswalk_WebApp_openfile_docx_windows.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<!--
+Copyright (c) 2013 Intel Corporation.
```
Comment (Honry) [inline wrt/wrt-packagemgtmanu-android-tests/packagemgtmanu/Crosswalk_WebApp_openfile_docx_windows.html]: Please update the copyright and code format with latest one.

File: wrt/wrt-packagemgtmanu-android-tests/packagemgtmanu/Crosswalk_WebApp_openfile_docx_windows.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      <strong>Pre-condition:</strong>
+    </p>
+    <ul>
+      <li>Install WPS or other thirdpart app to open docx/png/pdf file</li>
+    </ul>
+    <p>
+      <strong>Test steps:</strong>
+    </p>
+    <ol>
+      <li>Put a file named "test.docx" to device directory "/C:/"</li>
```
Comment (Honry) [inline wrt/wrt-packagemgtmanu-android-tests/packagemgtmanu/Crosswalk_WebApp_openfile_docx_windows.html]: "/C:/"->"C:\"

File: wrt/wrt-packagemgtmanu-android-tests/packagemgtmanu/Crosswalk_WebApp_openfile_docx_windows.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      <li>Install "web_openfile_thirdpart_tests" app</li>
+      <li>Launch the app</li>
+      <li>Click the "Open docx file in the third part app(/C:/test.docx)" link</li>
```
Comment (Honry) [inline wrt/wrt-packagemgtmanu-android-tests/packagemgtmanu/Crosswalk_WebApp_openfile_docx_windows.html]: /C:/test.docx -> C:\test.docx

**Comment by jennycao** [comment]
LGTM

**Pull request merged**